test/e2e/logs_test.go

```diff
@@ -468,11 +468,11 @@ var _ = Describe("Podman logs", func() {
 			testPod.WaitWithDefaultTimeout()
 			Expect(testPod).To(Exit(0))
 
-			log1 := podmanTest.Podman([]string{"run", "--log-driver", log, "--name", containerName1, "-d", "--pod", podName, BB, "/bin/sh", "-c", "echo log1"})
```

Comment:
the test above has the same bug, and while you are at it could you please remove the unnecessary `"/bin/sh", "-c"`
and just exec `"echo", "log1"`